migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
What is this describing? Maybe you need to add more information. And remember the trailing new line